reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md

```diff
@@ -102,24 +101,19 @@ parameter attributes.
 
 This setting indicates whether the parameter is mandatory, that is, whether
 all commands that use this cmdlet must include this parameter. When the value
-is **"True"** and the parameter is missing from the command, Windows PowerShell
+is **"True"** and the parameter is missing from the command, PowerShell
```

Comment:
If it's being bolded, should it also have quotations?

Reply:
Fixed